shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceImpl.java

```diff
@@ -89,7 +89,7 @@ void syncUpdatesFromManager() {
                                                         failure -> failedToSendUpdateToManager(y, failure));
                                     }
                                     if (y.getStatus().equals(BridgeStatus.DELETION_REQUESTED)) { // Bridges to delete
-                                        y.setStatus(BridgeStatus.DELETED);
+                                        y.setStatus(BridgeStatus.DELETING);
```

Comment:
Should it be applied also to `fetchAndProcessProcessorsToDeployOrDelete` below?

Reply:
yes, thanks